### PR TITLE
Fix redundancy bug.

### PR DESCRIPTION
**Pull request by shivanshuraj1333**
This pull request resolves Issue #20.

### CODE REVIEW

File: scripts/dbm.py [hunk 1]
```diff
@@ -7,12 +7,6 @@
 from scripts.os_check import ADB, SEP, SUC, PERM
 from scripts.utils import OUTPUT
 
-try:
```
Comment (vivonk) [inline scripts/dbm.py:10]: Correct, good catch :)

**Review by vivonk — APPROVED** [review]
GTG

**Comment by shivanshuraj1333** [comment]
Thanks, @vivonk.

**Review by Wolf-Legend — APPROVED** [review]
Approved

**Pull request merged**